test/src/e2e.dynval/dv.n'.test.ts

```diff
@@ -136,4 +136,269 @@ describe("Dynamic Validator N -> N'", function() {
+            const margin = 1.1;
+            const [bettyNode, ...otherDynNodes] = allDynNodes;
+            {
+                const authors = (await stake.getPossibleAuthors(
+                    otherDynNodes[0].sdk
+                ))!.map(author => author.toString());
+                expect(authors).to.includes(alice.platformAddress.toString());
+                expect(authors).not.to.includes(
+                    betty.platformAddress.toString()
+                );
+                expect(authors.length).to.be.equals(8);
+            }
+
+            {
+                const seq = await otherDynNodes[0].sdk.rpc.chain.getSeq(
+                    faucetAddress
+                );
+                const tx = stake
+                    .createDelegateCCSTransaction(
+                        otherDynNodes[0].sdk,
+                        betty.platformAddress,
+                        5_000
+                    )
+                    .sign({
+                        secret: faucetSecret,
+                        seq,
+                        fee: 10
+                    });
+                const tx2 = stake
+                    .createRevokeTransaction(
+                        otherDynNodes[0].sdk,
+                        alice.platformAddress,
+                        4999
+                    )
+                    .sign({
+                        secret: faucetSecret,
+                        seq: seq + 1,
+                        fee: 10
+                    });
+                await otherDynNodes[0].sdk.rpc.chain.sendSignedTransaction(tx);
+                await otherDynNodes[0].sdk.rpc.chain.sendSignedTransaction(tx2);
+            }
+
+            await otherDynNodes[0].waitForTermChange(2, TERM_SECONDS * margin);
+
+            const blockNumber = await otherDynNodes[0].sdk.rpc.chain.getBestBlockNumber();
+            const termMetadata = await stake.getTermMetadata(
+                otherDynNodes[0].sdk,
+                blockNumber
+            );
+
+            {
+                const authors = (await stake.getPossibleAuthors(
+                    otherDynNodes[0].sdk
+                ))!.map(author => author.toString());
+                expect(authors).not.to.includes(
+                    alice.platformAddress.toString()
+                );
+                expect(authors).to.includes(betty.platformAddress.toString());
+                expect(authors.length).to.be.equals(8);
+            }
+
+            expect(termMetadata).not.to.be.null;
+            expect(termMetadata!.currentTermId).to.be.equals(2);
+            expect(termMetadata!.lastTermFinishedBlockNumber).to.be.lte(
+                blockNumber
+            );
+        });
+    });
+
+    describe("4. Revoke the validator + increase the deposit of a candidate who doesn’t have enough deposit", async function() {
+        // alice : Elected as a validator, but all delegated CCS is revoked.
+        //   Alice should be jailed.
```

Comment:
You need to fix the comment here. Alice would not be jailed.

Reply:
Thanks! I missed it.